tests/all/winch.rs

```diff
@@ -0,0 +1,41 @@
+use anyhow::Result;
+use wasmtime::*;
+
+#[test]
+fn compiles_with_winch() -> Result<()> {
+    let mut c = Config::new();
+
+    // DOIT: once we remove winch as a default feature before push, update the features for this
```

Comment:
This still needs to be done, only keeping `winch` as a default feature so `rust-analyzer` works properly. I need to look into how to enable flags for the LSP without needing them to be default